content/doc/administration/requirements/upgrade-java-to-17.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: documentation
+title: Upgrading Jenkins Java version from 11 to 17
+---
+
+There are a few details and steps to upgrade the JVM used to run Jenkins, more specifically from Java 11 to Java 17.
+
+If you're upgrading the JVM used to run Jenkins, particularly from Java 11 to Java 17, there are some details you should know and precautions you should take.
+
+.Upgrading Jenkins Java Version From 11 to 17 
+video::ZabUz6sl-8I[youtube,width=800,height=420]
+
+== Backing up Jenkins
+
+As with any upgrade, link:/doc/book/system-administration/backing-up/#jenkins_home[we recommend backing up] `JENKINS_HOME` and testing the upgrade with the backup before performing the upgrade on your production instance.
+
+== Upgrading Jenkins
+
+If you need to upgrade Jenkins as well as the JVM, we recommend that you:
+
+. Update all of your installed plugins.
+. Back up `JENKINS_HOME`.
+. Stop the Jenkins instance.
+. Upgrade the JVM on which Jenkins is running.
+** Use a package manager to install the new JVM.
+** Make sure the default JVM is the newly installed version.
+If it is not, run `systemctl edit jenkins` and set either the `JAVA_HOME` environment variable or the `JENKINS_JAVA_CMD` environment variable.
+. Upgrade Jenkins to the most recent version
+** How you upgrade Jenkins is dependent upon your original Jenkins installation method.
+** We recommend using the package manager of your system, such as `apt` or `yum`.
+. Validate the upgrade to confirm that all plugins and jobs are loaded.
+. Upgrade the required plugins.
+Refer to <<Upgrading Plugins>> for more information.
+
+Starting with the Jenkins 2.357 weekly release and Long Term Support release 2.361.1, Java 11 or Java 17 is required.
```

Comment:
Since they are reading the upgrade instructions for Java 17, I think we can safely remove the mention of the version history.
```suggestion.
```

Reply:
That makes sense @MarkEWaite, I'll adjust that now.